src/main/com/fulcrologic/rad/type_support/date_time.cljc

```diff
@@ -240,13 +241,14 @@
    [string? => inst?]
    (html-datetime-string->inst *current-zone-name* date-time-string))
   ([zone-name date-time-string]
-   [(? ::zone-name) string? => inst?]
+   [(? ::zone-name) string? => (? inst?)]
    (try
-     (let [z   (get-zone-id zone-name)
-           dt  (ldt/parse date-time-string)
-           zdt (ldt/at-zone dt z)
-           i   (zdt/to-instant zdt)]
-       (new-date (instant/to-epoch-milli i)))
+     (when-not (= "" date-time-string)
```

Comment:
if date-time-string is nil, then this will also throw. seems a shame to rely on the try/catch for that...even though it is still needed. I don't think this when-not helps anything.

Reply:
The incoming value from "empty" date input will be `""`. 

This function is used as `:string->model` function.

The current behavior is that when empty input is parsed to model, `ldt/parse` will throw error that it cannot parse `''`.

This is a fix, now `""` in input will correctly convert to `nil` in data model. If `nil` is input then it throws, but it did so before. I'll fix the spec.

Reply:
ok, thanks